ending_location_visitor.py

```diff
@@ -1,32 +1,223 @@
+"""
+Astroid Source:
+https://github.com/PyCQA/astroid/blob/master/astroid/transforms.py
+
+See class and method docstrings for explanations.
+"""
+
+from sys import exit
 from astroid.transforms import TransformVisitor
 import astroid
+import unittest
+import logging
+from setendings import *
+from inspect_ast import visit_astroid
+
+# Set the log level (DEBUG, ERROR, ...), and message format.
+logging.basicConfig(format='', level=logging.DEBUG)
+
+
+class EasyTransformVisitor(TransformVisitor):
```

Comment:
This class is not useful. Please remove it can just call `register_transform` inline.